src/model/control_desk.py

```diff
@@ -127,53 +137,69 @@ def encoder_position(self, position: int):
 class ColorDeskColumn(DeskColumn):
     def __init__(self, _id: str = None):
         super().__init__(_id)
-        self._color_h = 0.0
-        self._color_s = 0.0
-        self._color_i = 0.0
+        self.color: ColorHSI = ColorHSI(0, 0, 0)
 
     def _generate_column_message(self) -> proto.Console_pb2.fader_column:
         base_msg = self._generate_base_column_message()
-        base_msg.plain_color.hue = self._color_h
-        base_msg.plain_color.saturation = self._color_s
-        base_msg.plain_color.intensity = self._color_i
+        base_msg.plain_color.hue = self.color.hue
+        base_msg.plain_color.saturation = self.color.saturation
+        base_msg.plain_color.intensity = self.color.intensity
         return base_msg
 
     @property
-    def color(self) -> Tuple[float, float, float]:
-        return (self._color_h, self._color_s, self._color_i)
+    def color(self) -> ColorHSI:
+        """color of the colum"""
+        return self.color
 
     @color.setter
-    def color(self, h: float, s: float, i: float):
-        self._color_h = h
-        self._color_s = s
-        self._color_i = i
+    def color(self, color: ColorHSI):
+        self.color = color
         self.update()
 
 
 class FaderBank:
+    """TODO Docstring"""
+
     def __init__(self):
         self.columns = []
         self._pushed_to_device = False
 
     def add_column(self, col: DeskColumn):
+        """add a new colum"""
         self.columns.append(col)
 
     def _generate_bank_message(self) -> proto.Console_pb2.add_fader_bank_set.fader_bank:
         msg = proto.Console_pb2.add_fader_bank_set.fader_bank()
         for col in self.columns:
-            msg.cols.extend([col._generate_column_message()])
+            msg.cols.extend([col._generate_column_message()])  # TODO private Methode
         return msg
 
 
 class BankSet:
-
+    """TODO docstring"""
```

Comment:
"""This class represents a bank set.

A bank set is a set of banks, the user can switch between at a given time. Multiple bank sets can be linked to fish at the same time but only one may be active at the same time. Only the GUI may specify which bank set is active any given time, except for the event that the active bank set will be unlinked. In this case fish will enable the bank set with the next lower index.
"""